### PR TITLE
[13]styled productpage

### PR DESCRIPTION
**Pull request by Iyiola-stack**
## Task: `Styled the product page using css
desk1
mo1
mo2

- `Changes you made to given task`

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done. Please add space between each product related items on the mobile screen size. They are too close

File: submissions/peterIyiola/css/productpage.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    background-color: greenyellow;
+    padding: 5px;
+    justify-content: space-around;
+}
+.logo   {
+    height: 30px;
+    width: 40px;
+}
+
+.productpagelink {
```
Comment (ToluAkin) [inline submissions/peterIyiola/css/productpage.css:14]: @Iyiola-stack You can group 14-16. They have the same style.

File: submissions/peterIyiola/css/productpage.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    flex-wrap: wrap;
+}
+.grid-similaritems   {
+    display:block;
+    justify-items: center;
+    align-items: center;
+
+}
+.similaritem1   {
+    margin-left: 10%;
```
Comment (ToluAkin) [inline submissions/peterIyiola/css/productpage.css:153]: Use `px` instead of `%` @Iyiola-stack

File: submissions/peterIyiola/css/productpage.css [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+}
+.similaritem2   {
+    margin-left: 13%;
```
Comment (ToluAkin) [inline submissions/peterIyiola/css/productpage.css:156]: Use `px` instead of `%` @Iyiola-stack

File: submissions/peterIyiola/css/productpage.css [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+}
+.similaritem3   {
+    margin-right:50px;
+    margin-left: 14%;
```
Comment (ToluAkin) [inline submissions/peterIyiola/css/productpage.css:160]: Use `px` instead of `%` @Iyiola-stack

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Iyiola-stack** [comment]
Goodmorning,
Kindly help with my PR
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/414/commits

**Pull request closed without merging**